src/clipboard.rs

```diff
@@ -30,7 +30,7 @@ fn detect_copy_provider() -> Provider {
   // Wayland
   if std::env::var("WAYLAND_DISPLAY").is_ok() {
     if let Some(provider) =
-      check_prog("wl-copy", &["--foreground", "--type", "text/plain"])
```

Comment:
Thank you for the PR! What was the reason for removing this? If I'm not mistaken, there was a problem without that argument. But I can't remember for sure.

Reply:
with --foreground, mprocs process just hangs forever after copying (doesn't respond to user input).